api/src/libs/messaging.ts

```diff
@@ -13,64 +13,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-// Created by Jason Leach on 2020-09-09.
-//
 
 import { logger } from '@bcgov/common-nodejs-utils';
 import fs from 'fs';
 import path from 'path';
+import config from '../config';
 import DataManager from '../db';
-import { Contact } from '../db/model/contact';
+import { ProjectProfile } from '../db/model/profile';
+import { HumanAction } from '../db/model/request';
 import { BodyType, Message, SendReceipt } from '../libs/service';
 import shared from './shared';
+import { transformContacts } from './utils';
 
 export const enum MessageType {
   ProvisioningStarted = 0,
   ProvisioningCompleted,
+  EditRequestStarted,
+  EditRequestCompleted,
+  RequestApproval,
+  RequestRejected
 }
 
-export const contactsForProfile = async (profileId: number): Promise<Contact[]> => {
-
-  try {
-    const dm = new DataManager(shared.pgPool);
-    const { ContactModel } = dm;
-
-    return await ContactModel.findForProject(profileId);
-  } catch (err) {
-    const message = `Unable to fetch contacts for profile ${profileId}`;
-    logger.error(`${message}, err = ${err.message}`);
-
-    return [];
-  }
-}
-
-export const profileDetails = async (profileId: number): Promise<any> => {
-
-  try {
-    const dm = new DataManager(shared.pgPool);
-    const { ProfileModel } = dm;
-    return await ProfileModel.findById(profileId);
-  } catch (err) {
-    const message = `Unable to fetch profile ${profileId}`;
-    logger.error(`${message}, err = ${err.message}`);
-
-    return [];
-  }
-}
+const dm = new DataManager(shared.pgPool);
+const { ProfileModel, ContactModel, RequestModel} = dm;
 
-export const updateEmailContent = async (buff: string, to: string[], profile: any, contactNames: string[]): Promise<string> => {
+const updateEmailContent = async (buff: string, profile: ProjectProfile, contactDetails: any, request: any, humanAction: any): Promise<string> => {
   try {
     let emailContent: string;
+    
+    let humanActionComment: string;
+
+    if(humanAction !== undefined) {
```

Comment:
Wonder if you could do this to remove the `else`
```JavaScript
let humanActionComment = '';

if (typeof humanAction !== 'undefined) {
   humanActionComment = `Additional information: ${humanAction.comment}`;
}
```

Reply:
Good catch! Much cleaner